src/infextendedtime/comparison.jl

```diff
@@ -0,0 +1,16 @@
+Base.isfinite(x::InfExtendedTime) = x.flag == finite && isfinite(x.finitevalue)
+Base.isinf(x::InfExtendedTime) = isposinf(x) || isneginf(x)
+Base.:(==)(x::InfExtendedTime, y::InfExtendedTime) = (isfinite(x) && isfinite(y)) ? x.finitevalue == y.finitevalue : x.flag == y.flag
+Base.hash(x::InfExtendedTime, h::UInt) = isfinite(x) ? hash(x.finitevalue, h) : hash(x.flag, h)
```

Comment:
Currently wouldn't support hashing different infinities (e.g. `InfExtended` vs. `InfExtendedTime`) to the same result. I'm not sure what the desired behaviour is.

Reply:
Good point.

Reply:
I'd like the hash to simply defer to whatever the true value is, i.e. the `finitevalue` or `PosInf` or `NegInf`. It's fine that the infinite date has the same hash as the infinite number.